web/src/routes/MyPages/ProfilePage.tsx

```diff
@@ -153,7 +146,14 @@ const formatPhoneNumber = (value: string): string => {
 export default function MyPage() {
   const [theme, toggleTheme] = useTheme();
   const navigate = useNavigate();
-  const [state, dispatch] = useReducer(profileReducer, initalState);
+  const [passwordDisabled, setPasswordDisabled] = useState(false);
+  const [phoneNumberDisabled, setPhoneNumberDisabled] = useState(false);
+  const [addressDisabled, setAddressDisabled] = useState(false);
+  const [titleAlert, setTitleAlert] = useState('');
+  const [textAlert, setTextAlert] = useState('');
+  const [isAlertModal, setIsAlertModal] = useState(false);
```

Comment:
Okay `isAlertModal`. "Well, guess what it is~? Ha ha this is AlertModal~~~!"?

Nope. `showAlertModal`.